content/en/synthetics/browser_tests/actions.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Browser Test Actions
+kind: documentation
+description: Record Actions for a Synthetics Browser Test
+aliases:
+- /synthetics/browser_tests
+further_reading:
+- link: "synthetics/browser_tests/advance_options"
+  tag: "Documentation"
+  text: "Learn how to configure advance options for Actions"
+---
```

Comment:
Global note just to make sure we don't forget about removing the 3 sections about advanced options on this page

Reply:
Just removed 👍